packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -35,8 +36,9 @@ async function collectAccounts(ix, accounts, last, depth = 3) {
     if (Array.isArray(ax)) {
       await collectAccounts(ix, ax, old, depth - 1)
     } else {
+      if (ax.addr) ax.addr = sansPrefix(ax.addr)
       if (ax.addr != null && ax.keyId != null) {
-        ax.tempId = `${ax.addr}-${ax.keyId}`
+        ax.tempId = `${withPrefix(ax.addr)}-${ax.keyId}`
```

Comment:
Must standardize all tempId to withPrefix in order to maintain backwards compatibility/in order for new sansPrefix code to work

Reply:
Want to put this into a function? I see it repeated below.